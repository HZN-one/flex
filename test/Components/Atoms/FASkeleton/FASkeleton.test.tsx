/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";

import { FASkeleton } from "@Atoms";

import { render } from "@testing-library/react";

describe("FASkeleton", () => {
  it("should match snapshot", () => {
    const { container } = render(<FASkeleton testID="skeleton-example" />);
    expect(container).toMatchSnapshot();
  });
});
