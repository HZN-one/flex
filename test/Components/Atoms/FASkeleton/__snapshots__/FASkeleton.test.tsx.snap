// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FASkeleton should match snapshot 1`] = `
<div>
  <span
    class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1n7gpw1-MuiSkeleton-root"
    data-testid="skeleton-example"
  />
</div>
`;
